static/templates/giphy_picker.hbs

```diff
@@ -0,0 +1,12 @@
+<div class="popover" id="giphy_grid_in_popover">
+    <div class="arrow"></div>
+    <div class="popover-inner">
+        <div class="search-box">
+            <input type="text" id="giphy-search-query" class="search-query" placeholder="{{t 'Search GIFs' }}">
+        </div>
+        <div class="popover-content"></div>
+        <div class="popover-footer">
+            <img src="/static/images/GIPHY_attribution.png" alt="{{t 'GIPHY attribution' }}">
+        </div>
+    </div>
+</div>
```

Comment:
I haven't investigated why this is broken, but keyboard accessibility of tabbing to the button to open this UI widget doesn't work.  It seems to me like an important feature as part of our general commitment to making everything in Zulip keyboard-accessible, so we should at least fix the tab index situation there.

(And possibly also see if we can make keyboard navigation work once the picker is open, if it doesn't already -- haven't tried the end-to-end test since the first step doesn't work).

Reply:
Added a keyboard accessibility and filed https://github.com/Giphy/giphy-js/issues/174